codespell_lib/data/dictionary.txt

```diff
@@ -6839,7 +6845,6 @@ conveting->converting
 convetion->convention
 convets->converts
 convexe->convex
```

Comment:
Probably makes sense to also suggest `convexes` here? See #1426

Reply:
added.